### DISCUSSION TITLE
Installation issue: no wmake

### DISCUSSION THREAD

**Discussion by DeMilder** [ITHACA-FV/ITHACA-FV#472] [General]
Hi, I am new to OpenFOAM, to ROM and also to Github. I would like to try this package out. However, I encountered an installation issue. When I run the last step of the installation instructions:
`sudo ./Allwmake`
(I needed to add sudo to give permissions)
I obtain the error
`Error: no wmake - is OPENFOAM enviroment active?`

Can somebody explain what this error message means and how to tackle it?

Thanks in advance!

**Comment by giovastabile** [comment]
Hi @DeMilder you need to source the openfoam bashrc file and the ithaca-fv bashrc file. In order to avoid using sudo you can clone ITHACA-FV in a folder in which you have permits (in your home for example)

`source your_openfoam_folder/etc/bashrc`
`source your_ithacafolder/etc/bashrc`

after that you can run ./Allwmake as usual

**Reply by DeMilder** [reply]
Hi @giovastabile, thank you for your response! I have now installed ITHACA-FV in my home/Programs directory and the ./Allwmake command worked! However, when I now try
'milder@milder-VirtualBox:~/Programs/ITHACA-FV/tutorials/CFD/01POD$ perform_POD'
I get that there is no perform_POD:
'perform_POD: command not found'
Is this because ITHACA-FV is not installed in the ${FOAM_APP}/utilities folder?

**Reply by giovastabile** [reply]
perform_POD is an application so if you want to compile applications as well you should use:
`./Allwmake -a`
as reported in the README file
